src/trackers/ANIRENA.py

```
@@ -0,0 +1,383 @@
+                    res_data = response.json()
+                    torrents = res_data.get('torrents', [])
+                    for t in torrents:
+                        # Simple name matching or info_hash matching if we had it
+                        # Since we don't have the info_hash yet (torrent not created or hash not in meta),
+                        # we rely on name matching.
+                        dupes.append({
+                            'name': t.get('title'),
+                            'size': t.get('size_fmt'),
+                            'link': f"{self.base_url}/torrent/{t.get('id')}"
+                        })
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)